configs/components/ruby.rb

```diff
@@ -17,6 +17,11 @@
   pkg.replaces 'pe-ruby-ldap'
   pkg.replaces 'pe-rubygem-gem2rpm'
 
+  pkg.conflicts 'pe-puppet-enterprise-release', '2016.3.0.0'
+  pkg.conflicts 'pe-installer', '2016.1.0.70'
```

Comment:
This one is interesting. PE 2016.2.1 shipped with `pe-installer` build `2016.1.0.47`.  Current pre-release builds of PE have `pe-installer` build `2016.1.0.70`.